internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -444,8 +444,7 @@ func (r *ISBServiceRolloutReconciler) isISBServiceUpdating(ctx context.Context,
 
 func (r *ISBServiceRolloutReconciler) GetPipelineList(ctx context.Context, rolloutNamespace string, rolloutName string) (*unstructured.UnstructuredList, error) {
 	gvk := schema.GroupVersionKind{Group: common.NumaflowAPIGroup, Version: common.NumaflowAPIVersion, Kind: common.NumaflowPipelineKind}
-	return kubernetes.ListResources(ctx, r.client, gvk,
-		client.InNamespace(rolloutNamespace),
+	return kubernetes.ListResources(ctx, r.client, gvk, rolloutNamespace,
```

Comment:
this function now always checks the namespace passed in instead of passing it in as a LabelSelector, which would be easy for a caller to accidentally leave out (also creates more consistency with the other functions in `unstructured_util.go`, which take namespace as parameter). I don't think we have a need currently for listing resources across all namespaces. I guess if we do in the future then we can change this or create separate functions that are more explicit.